src/main/java/step2/model/Rank.java

```diff
@@ -0,0 +1,34 @@
+package step2.model;
+
+import java.util.Arrays;
+
+public enum Rank {
+	FIRST(6, 2_000_000_000),
+	SECOND(5, 30_000_000),
+	THIRD(5, 1_500_000),
+	FOURTH(4, 50_000),
+	FIFTH(3, 5_000),
+	MISS(0, 0);
+
+	private int countOfMatch;
+	private int winningMoney;
+
+	Rank(int countOfMatch, int winningMoney) {
+		this.countOfMatch = countOfMatch;
+		this.winningMoney = winningMoney;
+	}
+
+	public int getCountOfMatch() {
+		return countOfMatch;
+	}
+	public int getWinningMoney() {
+		return winningMoney;
+	}
+
+	public static Rank of(int countOfMatch) {
+		return Arrays.stream(Rank.values()).filter(
+			rank -> rank.getCountOfMatch() == countOfMatch).findFirst().orElse(MISS);
+
```

Comment:
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`를 준수해보면 좋을 것 같아요.
```suggestion
		return Arrays.stream(values())
			.filter(rank -> rank.getCountOfMatch() == countOfMatch)
			.findFirst()
			.orElse(MISS);
```